content/en/docs/v3.4/quickstart.md

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
The same changes suggested for the `next` page should be done here.

Reply:
Are the `next` and `v3.4` pages identical?

Reply:
> Are the `next` and `v3.4` pages identical?

They should be.

Reply:
Thanks for checking. (I don't like this duplication, but I'm unsure what to do about it.)